src/types/types.ts

```diff
@@ -760,6 +760,22 @@ export interface GetFloorPriceResponse {
   readonly looksRare: FloorPriceMarketplace | FloorPriceError;
 }
 
+/**
+ * Information about the rarity of an NFT's attribute in the specified collection.
+ *
+ * @public
+ */
+export interface NftAttributeRarity {
+  /** Name of the NFT's attribute. */
+  value: string;
+
+  /** The type of NFT attribute. */
+  trait_type: string;
```

Comment:
Please rename to `traitType` in order to be consistent with JS style.

Reply:
On it 👌🏼